system/templates/rolls/roll-graphic.hbs

```diff
@@ -1,17 +1,19 @@
-<div class="dice-roll ironsworn-roll">
-
-  <div class="dice-formula">
-    <p>
+<article class="dice-roll ironsworn-roll">
+  <section class="dice-formula">
```

Comment:
Not sure if you've tested this, but we lost some vertical whitespace here.

Before:

![CleanShot 2022-08-21 at 06 55 04](https://user-images.githubusercontent.com/39902/185794355-51f7d6c0-4844-4941-b015-c26d330db5d4.jpg)


After:

![CleanShot 2022-08-21 at 06 52 42](https://user-images.githubusercontent.com/39902/185794309-68758b94-37aa-4989-8f5d-f6a959b42a31.jpg)

Also, the action score total is still missing, and the outcome is still mistranslated.

Reply:
yeah, the styling is often the absolute *last* thing i look at, haha. the semantic content of HTML is much more rigid than CSS, which is ultimately just rendering parameters. the CSS styling is definitely important, but should be informed by the semantics and not the other way around. practically speaking, most non-form HTML elements are just restylable boxes anyway, so they're roughly interchangeable as far as CSS is concern - they're about what the content *is*, rather than a vessel for applying styles *per se*.

the bad `Strong_hit` string, on the other hand, is pure goof on my part ;)